### ISSUE TITLE
Comparing Different models

### ISSUE THREAD

**Issue by mohammadshahabuddin** [junyanz/pytorch-CycleGAN-and-pix2pix#1231]
Hi, 
Could you please tell me how you compared different models? Did you use the same learning rate, number of epochs, Number of decay epochs, image size, optimizer among all models? Also, did you collect test results using the final saved generator or did you use the best results testing all saved generators at different epochs?
